components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -149,33 +141,30 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
         this.attributes = attributes;
       });
 
-    // handle rule creation/update success
-    combineLatest([
-      this.store.select(createStatus),
-      this.store.select(updateStatus)
-    ]).pipe(
```

Comment:
Separated create and update cases to (a) have simpler code and (b) avoid problems with the filter allowing one through when we might want the other one through.